sim/druid/swipe.go

```diff
@@ -62,18 +50,31 @@ func (druid *Druid) registerSwipeBearSpell() {
 			IgnoreHaste: true,
 		},
 
-		DamageMultiplier: 1 + 0.1*float64(druid.Talents.SavageFury),
+		DamageMultiplier: baseMultiplier + 0.1*float64(druid.Talents.SavageFury),
 		ThreatMultiplier: SwipeThreatMultiplier,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
 			for idx := range results {
+				dotBonusCrit := 0.0
+				if druid.LacerateBleed.Dot(target).GetStacks() > 0 {
+					dotBonusCrit = druid.FuryOfStormrageCritRatingBonus
+				}
+
+				spell.BonusCritRating += dotBonusCrit
 				results[idx] = spell.CalcDamage(sim, target, baseDamage, spell.OutcomeMeleeSpecialHitAndCrit)
+				spell.BonusCritRating -= dotBonusCrit
+
 				target = sim.Environment.NextTargetUnit(target)
 			}
 
 			for _, result := range results {
 				spell.DealDamage(sim, result)
 			}
+
+			if druid.HasRune(proto.DruidRune_RuneHelmGore) && sim.Proc(0.15, "Gore") {
```

Comment:
Can you cache the rune check as a variable outside of the spell definition for performance?